src/gui/transferlistsortmodel.cpp

```diff
@@ -146,14 +144,13 @@ bool TransferListSortModel::lessThan(const QModelIndex &left, const QModelIndex
             // 1. Active torrents at the top
             // 2. Seeding torrents at the bottom
             // 3. Torrents with invalid ETAs at the bottom
-
-            const bool isActiveL = TorrentFilter::ActiveTorrent.match(model->torrentHandle(model->index(left.row())));
-            const bool isActiveR = TorrentFilter::ActiveTorrent.match(model->torrentHandle(model->index(right.row())));
+            const bool isActiveL = TorrentFilter::ActiveTorrent.match(model->torrentHandle(left));
```

Comment:
Isn't this the case I described? `left` is index of **TransferListSortModel** but `model` is instance of **TransferListModel**.

Reply:
https://doc.qt.io/qt-5/qsortfilterproxymodel.html#lessThan
>Note: The indices passed in correspond to the source model.

I changed to `model->torrentHandle(mapToSource(left))` and it starts giving warnings in console: `QSortFilterProxyModel: index from wrong model passed to mapToSource`.

Reply:
Hmmm... maybe leave a comment:
>Note: The indices passed in correspond to the source model.

Reply:
ok, done.